src/components/commons/Footer.tsx

```diff
@@ -19,6 +19,7 @@ export default function Footer(props: { fixedPages: Array<string> }) {
 
   return (
     <Box
+      component="footer"
```

Comment:
This attribute is duplicated

Reply:
done here 1ca79550c720818e20f1ea0006f87bedc35e0924